selfdrive/ui/ui

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 export LD_LIBRARY_PATH="/system/lib64:$LD_LIBRARY_PATH"
+export QT_PLUGIN_PATH="../../phonelibs/qt-plugins/$(uname -m)"
```

Comment:
should we move this stuff to process_config.py?

Reply:
Yes. But maybe something for a standalone PR.

Reply:
@deanlee interested in fixing this up?